src/common/helper.js

```diff
@@ -440,6 +441,130 @@ async function createResource (challengeId, memberHandle, roleId) {
   return res || false
 }
 
+/**
+ * Create Project
+ * @param {String} name The name
+ * @param {String} description The description
+ * @param {String} type The type
+ * @param {String} token The token
+ * @returns 
+ */
+async function createSelfServiceProject (name, description, type, token) {
+  const projectObj = {
+    name,
+    description,
+    type
+  }
+  const url = `${config.PROJECTS_API_URL}`
+  const res = await axios.post(url, projectObj, {headers: {Authorization: `Bearer ${token}`}})
+  return _.get(res, 'data.id')
+}
+
+/**
+ * Get project payment
+ * @param {String} projectId the project id
+ */
+ async function getProjectPayment (projectId) {
+  const token = await getM2MToken()
+  const url = `${config.CUSTOMER_PAYMENTS_URL}`
+  const res = axios.get(url, {
+    headers: { Authorization: `Bearer ${token}` },
+    params: {
+      referenceId: projectId
```

Comment:
@ThomasKranitsas it's better to filter by both `reference=project` and `referenceId=projectId` to avoid situation when there are several objects with the same id, for example there could project and phase with the same id.